src/main/scala/cromwell/binding/WdlBinding.scala

```diff
@@ -115,6 +114,11 @@ object WdlBinding {
     }
   }
 
+  /* All MemberAccess ASTs that are not contained in other MemberAccess ASTs */
+  def findTopLevelMemberAccesses(expr: AstNode) = WdlBinding.findAstsWithTrail(expr, "MemberAccess").filter {
+    case (k, v) => !(v map {case a:Ast => a.getName} contains "MemberAccess")
```

Comment:
You can probably do this w/ a for comp. If not, start breaking things down into funcs so you don't have multiple cased higher order functions in a line